src/Php80/Php80.php

```diff
@@ -88,17 +88,23 @@ public static function preg_last_error_msg(): string
         }
     }
 
-    public static function str_contains(string $haystack, string $needle): bool
+    public static function str_contains(?string $haystack, ?string $needle): bool
     {
+        // Cast to string to avoid deprecation notices
+        if (null === $needle)
+        {
```

Comment:
This needs fixing - I'll sort after the tests have run in case there's another error I don't know about.